src/nat/llm/huggingface_llm.py

```
@@ -0,0 +1,289 @@
+            return chunks
+
+        try:
+            loop = asyncio.get_event_loop()
+        except RuntimeError:
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
+
+        chunks = loop.run_until_complete(_collect())
+        yield from chunks
+
+    async def astream(self, messages, **kwargs):
```

Ruff (0.14.8)

Unused method argument: `kwargs`

(ARG002)